gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -576,11 +576,63 @@ notifiers:
 
 # Cockpit
 #cockpit:
-# Specify the URL to cockpit instance. Default is the Gravitee SAAS instance
+#  enabled: false
+## Specify the URL to cockpit instance. Default is the Gravitee SAAS instance
 #  url: https://cockpit.gravitee.io
-# Specify the auth path used to authenticate from cockpit. {token} placeholder can be used and will be replaced by cockpit token.
+## Specify the auth path used to authenticate from cockpit. {token} placeholder can be used and will be replaced by cockpit token.
 #  auth:
-#    path: /management/auth/cockpit?token={token}
+#    path: /auth/cockpit?token={token}
+
+#installation:
+#  type: standalone # Could be either standalone, multi-tenant; Default is standalone.
+#  # Specify the URL of Management API of this instance, mandatory if you want to connect it to Cockpit
+#  api:
+#    proxyPath:
+#      management: ${http.api.management.entrypoint}
+#      portal: ${http.api.portal.entrypoint}
+#  standalone:
+#    api:
+#    # Specify the URLs of Management API, mandatory if you want to connect it to Cockpit with a standalone installation
+#      url: http://localhost:8083
+#    # Specify the URL of Console UI of this instance, mandatory if you want to connect it to Cockpit with a standalone installation
+#    console:
+#      url: http://localhost:3000
+#      # If you have multi console instances targeting different organizations, you can configure the section as bellow instead:
+#      urls:
+#         - orgId: DEFAULT
+#           url: http://localhost:3000
+#         - orgId: example
+#           url: http:/localhost:3001
+#    # Specify the URL of Portal UI of this instance
+#    portal:
+#      url: http://localhost:4100
+#      # If you have multi portal instances targeting different environments, you can configure the section as bellow instead:
+#      urls:
+#         - envId: DEFAULT
+#           url: http://localhost:4100
+#         - envId: example
+#           url: http:/localhost:4101
+#  multi-tenant:
+#    # Specify the access points of your installation, mandatory if you want to connect it to Cockpit with a multi-tenant installation
+#    # You can use template variable such as {account}, {organization} or {environment}
+#    accessPoints:
+#      organization:
```

Comment:
could we have a gateway in the organization ? Because today it's possible to link a gateway to a whole org.

Reply:
you can link a gateway to on orga? Where can you do that? 
Regarding the feature itself the gateway URL must be environment related not organization related to allow restrictedDomains

Reply:
https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/config/gravitee.yml#L541

Reply:
Everything here is only used by APIM, not the gateway. If you have defined a orga or an env in the gateway config, the gateway will filter the API based on that, nothing else.

Reply:
Thanks to our discussion, I finally understood 😅
It's ok 😉